CONTRIBUTING.md

```diff
@@ -33,6 +33,7 @@ wait for feedback from the developers.
 
 To ensure your PR won't be rejected please make sure that:
 
+* All CI tests for PR are passed, do not assign reviewers before got test results.
```

Comment:
If this was here before, it hadn't really sunk in for me-- what is the reasoning for saying "do not assign reviewers before got test results"?  (and at the least the grammar should be corrected to "...before getting...").

Reply:
Should I remove this line or only improve message?

Reply:
> Should I remove this line or only improve message?

Well, I was wanting to actually talk out the idea, not necessarily just remove the line.  I suppose the thinking is that until it passes the CI tests, it could be considered slightly premature to ask for a review, since the CI tests might mandate changes.  But I am inclined to think that there are already plenty of things besides CI tests that could cause mid-PR changes, like reviews from other people, or just the OP decides to change something or add more commits.   So it seems to me that just like now, it should be fine for people to assign reviewers when they make the PR.

So if you feel strongly about it, let's continue the discussion, otherwise, yes, let's take that line out.

Reply:
Removed, rebased.